search-app/resources/templates/base.html

```diff
@@ -0,0 +1,35 @@
+{% extends "templates/structure.html" %}
+
+{% block title %}CMR Search{% endblock %}
+
+{% block top-nav %}
+<div class="masthead clearfix">
+  <div class="inner">
+    <h3 class="masthead-brand"><a href="/">CMR</a></h3>
+    <ul class="nav masthead-nav">
+      <li><a href="/site/search_api_docs.html">API</a></li>
```

Comment:
This is now fixed, pending proper setting of `public-conf` in dev-system and integration tests.